src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -79,13 +79,14 @@ <h2 class="kuiTitle kuiVerticalRhythm">
 
   <dashboard-grid
     ng-show="!hasExpandedPanel()"
-    on-panel-removed="onPanelRemoved"
-    dashboard-view-mode="dashboardViewMode"
     panels="panels"
-    save-state="saveState"
-    toggle-expand="toggleExpandPanel"
+    is-full-screen-mode="!chrome.getVisible()"
+    dashboard-view-mode="dashboardViewMode"
+    get-embeddable-handler="getEmbeddableHandler"
+    get-container-api="getContainerApi"
```

Comment:
switched to getContainerApi instead of passing containerApi in directly to avoid circular references.